mavros_extras/scripts/mavteleop

```diff
@@ -77,6 +99,8 @@ rc_channels = {
     'throttle': RCChan('throttle', 2, 0.0)
 }
 
+rc = OverrideRCIn() # must be global to retain mode value
```

Comment:
The RC message needs to be a global variable in this case, to store the last mode selected using the gamepad as the buttons are immediately released (unlike the flight controller). However it can be moved inside `rc_override_control()` to minimize scope.